### PR TITLE
feat: added funders component

### LINKED ISSUE #45: Component: Footer / Funders
Funders component.

### PR DESCRIPTION
**Pull request by TeddTech**
resolves #45

### CODE REVIEW

File: components/SocialMedia.vue [hunk 1]
```diff
@@ -1 +1,7 @@
-<template><div /></template>
+<template>
+	<div>
+		<p>Funded by</p>
+		<img to="">
```
Comment (greatislander) [inline components/SocialMedia.vue]: Use config data for this.

File: components/SocialMedia.vue [hunk 2]
```diff
@@ -1 +1,7 @@
-<template><div /></template>
+<template>
+	<div>
```
Comment (greatislander) [inline components/SocialMedia.vue]: Give this `div` a unique class.

**Comment by TeddTech** [comment]
should I include a funders object property for `alt` attribute of the logo 

```html
			<li v-for="f in funders" :key="f.id">
				<nuxt-link :to="f.href">
					<img :src="f.logo" :alt="f.logoAlt">
					{{ f.title }}
				</nuxt-link>
			</li>
```

**Comment by greatislander** [comment]
@TeddTech Yes! Great idea!

File: components/SocialMedia.vue [hunk 3]
```diff
@@ -1,7 +1,23 @@
 <template>
-	<div>
-		<p>Funded by</p>
-		<img to="">
-		<img to="">
+	<div class="socila-media">
```
Comment (greatislander) [inline components/SocialMedia.vue]: Typo in this class.

Comment (TeddTech) [reply components/SocialMedia.vue]: Me and that word do not get along

Comment (TeddTech) [reply components/SocialMedia.vue]: all done

**Pull request merged**